From 8fe80db57c97a1fbdf38ae0aa29a57a8042c802c Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Tue, 6 Mar 2018 16:20:12 +0530
Subject: [PATCH 0326/1852] drm: xlnx: mixer: remove deprecated
 dmaengine_terminate_all

commit e052db82de04e829684336090a940c055c01710e from
https://github.com/Xilinx/linux-xlnx.git

Replacing dmaengine_terminate_all with dmaengine_terminate_sync,
as the former one is deprecated.

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_mixer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_mixer.c b/drivers/gpu/drm/xlnx/xlnx_mixer.c
index 8911cf676717..d043980f53ee 100644
--- a/drivers/gpu/drm/xlnx/xlnx_mixer.c
+++ b/drivers/gpu/drm/xlnx/xlnx_mixer.c
@@ -1268,7 +1268,7 @@ static void xlnx_mix_plane_dpms(struct drm_plane *base_plane, int dpms)
 		/* stop dma engine and release descriptors */
 		for (i = 0; i < XVMIX_MAX_NUM_SUB_PLANES; i++) {
 			if (plane->dma[i].chan && plane->dma[i].is_active) {
-				dmaengine_terminate_all(plane->dma[i].chan);
+				dmaengine_terminate_sync(plane->dma[i].chan);
 				plane->dma[i].is_active = false;
 			}
 		}
-- 
2.31.1

